app/experimenter/experiments/models.py

```diff
@@ -330,9 +336,16 @@ def generate_normandy_slug(self):
         if not self.bugzilla_id:
             raise ValueError(error_msg.format(field="Bugzilla ID"))
 
+        if self.firefox_max_version:
+            version_string = (
+                f"{self.firefox_min_version}-{self.firefox_max_version}"
+            )
+        else:
+            version_string = self.firefox_min_version
```

Comment:
It's a tiny stylistic thing, but I prefer to write things like this as

    thing = default
    if something:
        thing = special_case

The way you've written it is valid Python but it always irks me to introduce a variable inside an if statement, becuase you could sometimes forget a case and then suddenly wind up referring to a varaible that wasn't declared.  But it's just a preferential thing.

Reply:
ok I'll change that